docs/advanced-config-options.md

```diff
@@ -0,0 +1,277 @@
+# SignalFx Smart Agent Advanced Configuration Options
+
+
+__See [Overview](https://docs.signalfx.com/en/latest/integrations/agent/overview.html) for simplified Smart Agent command-line installation on a single host.__
+
+## Advanced Installation on a Single Host
```

Comment:
This doesn't make sense being in a file called "Advanced Configuration Options".